### PR TITLE
Add: kubectl and helm binaries

### PR DESCRIPTION
**Pull request by dzirg44**
Added: 
- kubectl
- helm

### CODE REVIEW

**Comment by binlab** [comment]
You should use a separate path (dir) like `docker-awscli` in our case it can be e.g. `kubectl-helm`.
For this has two reasons:
1. for build custom image with separate tag
2. for indicating which tools included to this image (e.g. not all need this tools)

**Comment by binlab** [comment]
Please note, base `jnlp/Dockerfile` not include `docker` and `awscli`

**Review by binlab — CHANGES_REQUESTED** [review]
You should use a separate path (dir) like `docker-awscli` in our case it can be e.g. `kubectl-helm`.
For this has two reasons:

for build custom image with separate tag
for indicating which tools included to this image (e.g. not all need this tools)

Please note, base `jnlp/Dockerfile` not include `docker` and `awscli`

File: jnlp/Dockerfile [hunk 1]
```diff
@@ -44,4 +44,4 @@ WORKDIR ${HOME}
 
 USER ${USER}
 
-ENTRYPOINT ["jenkins-agent"]
+ENTRYPOINT ["jenkins-agent"]
```
Comment (binlab) [inline jnlp/Dockerfile]: Please discard changes for line break

**Comment by dzirg44** [comment]
changes created

**Review by binlab — CHANGES_REQUESTED** [review]
Please add some documentation for README file (or just info about new future tag)

**Comment by dzirg44** [comment]
Added some information to the README.md

**Comment by binlab** [comment]
LGTM

**Review by binlab — APPROVED** [review]

**Pull request merged**